components/Sections/NewHeader/index.module.scss

```diff
@@ -0,0 +1,158 @@
+%navItemLi {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  margin-left: var(--space-16);
+  white-space: nowrap;
+
+  @media (max-width: 900px) {
+    display: flex;
+    height: 42px;
+    margin-left: 0;
+
+    &:not(:last-child) {
+      margin-right: 0.5rem;
```

Comment:
Do we have any space variable with similar value? (Not same, but similar)

Reply:
Exactly, not. Within a range of 2-3 pixels yes.

Reply:
That's fine for me, could we update?